### DISCUSSION TITLE
issue in running the command

### DISCUSSION THREAD

**Discussion by deepanshsinghal** [projectdiscovery/httpx#775] [Q&A] [answered]
i have installed httpx by clonning the repository. i dont know why this issue caused in my system. can you help me with the process.

i am using this command :
subfinder -d tesla.com | httpx -status-code
[WRN] Use with caution. You are responsible for your actions
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[WRN] By using subfinder, you also agree to the terms of the APIs used.

output:
[INF] Enumerating subdomains for tesla.com
Usage: httpx [OPTIONS] URL

Error: No such option: -s

**Comment by forgedhallpass** [comment]
Are you sure you are using our `httpx`?

Please see this discussion:
https://github.com/projectdiscovery/httpx/discussions/774#discussioncomment-3673676

**Comment by deepanshsinghal** [comment]
What purpose is your httpx for??

**Comment by forgedhallpass** [comment, marked as answer]
> What purpose is your httpx for??

You are on the ProjectDiscovery's `httpx` GitHub project. Considering this and that you want to use the `-status-code` flag, this is the tool that you were looking for, but there is also another project called [httpx](https://www.python-httpx.org/). You probably have both on your system.

You either need to:
* create a soft link to one of the higher priority directories within the `PATH` that points to the executable you want to use by default
* use the absolute path when starting ours
* delete the other tool
* update the order of your `PATH` variable

You can use the `which httpx` command to verify. When installing a go tool using the `go install` command, it is saved under `$GOPATH/bin`.

As mentioned before, see [this](https://github.com/projectdiscovery/httpx/discussions/774#discussioncomment-3673676) comment.

**Reply by victorcwb** [reply]
thanks my friends, resolved mi problem

**Reply by dreamdl** [reply]
Good, I used the whereis command to find httpx, then delete it all, and then execute the install command again. Although it doesn't seem to have been downloaded, I found it is OK.
